src/aui-scheduler/js/aui-scheduler-base.js

```diff
@@ -1447,6 +1466,19 @@ var SchedulerBase = A.Component.create({
 
                 instance.syncEventsUI();
             }
+        },
+
+        /**
+         * Set the `showToolbar` property - i.e., defines if the scheduler
+         * header will be displayed.
+         *
+         * @method _uiSetShowToolbar
+         * @protected
+         */
+        _uiSetShowToolbar: function() {
```

Comment:
This seems weird, a private function that only calls a public function, could you explain the why you had to create this function?